app/client/src/features/events/Questions/QuestionAuthor.tsx

```diff
@@ -41,7 +42,12 @@ export function QuestionAuthor({ fragmentRef, ...props }: QuestionAuthorProps) {
     );
     return (
         <CardHeader
-            title={<Typography>{authorData.createdBy?.firstName ?? 'Unknown User'}</Typography>}
+            avatar={
+                <Avatar>
+                    {authorData.createdBy?.firstName ? authorData.createdBy?.firstName[0] : 'U'}
```

Comment:
Copy/pasting comment from previous PR.

>This could potentially result in David undefined. I would put this into its own function so that way we can more robustly cover all cases.